### ISSUE TITLE
Add EPUB download option

### ISSUE THREAD

**Issue by Freso** [GenderDysphoria/GenderDysphoria.fyi#87]
[EPUB](https://en.wikipedia.org/wiki/EPUB) is a much more accessible format for ebook readers and mobile devices than PDFs (most of the time anyway), so it would be really nice to have an option for downloading as that.

I tried looking through the source to see how the PDF was made, since the more efficient path would be to automate the generation of the EPUB, but I didn’t find anything. Is the PDF manually updated? If so, I can probably make a commandline command or a small Python script that will compile the EPUB (pandoc should be able to deal with the brunt of it) on command.

**Comment by ErdaradunGaztea** [comment]
Just a small thought — perhaps we could utilize Github Actions to run potential document-compiling scripts?

**Comment by Twipped** [comment]
Yes, I had been making the PDF manually by printing to PDF. I would LOVE a github action to handle that, if somebody wants to spearhead it.

**Comment by gjvnq** [comment]
Look at [how I implemented WARC generation](https://github.com/gjvnq/GenderDysphoria.fyi/blob/pt-translation/gulp/offline.js). This might be able to be adapted into generating an EPUB.

As for generating PDF, one option would be using something like [Chrome Puppeteer](https://developers.google.com/web/tools/puppeteer).
